packages/third-parties/bullmq/src/BullMQModule.ts

```diff
@@ -1,7 +1,8 @@
 import {BeforeInit, DIContext, runInContext} from "@tsed/common";
 import {Constant, InjectorService, Module} from "@tsed/di";
+import {deepMerge} from "@tsed/core";
 import {getComputedType} from "@tsed/schema";
-import {Job, Queue, Worker} from "bullmq";
+import {Job, Queue, QueueOptions, Worker, WorkerOptions} from "bullmq";
```

Comment:
```suggestion
import {Job, Queue, type QueueOptions, Worker, type WorkerOptions} from "bullmq";
```

I think those could be type imports, we're not actually doing anything with them.